scripts/babel/proptypes-from-ts-props/index.js

```diff
@@ -137,10 +137,10 @@ function resolveOmitToPropTypes(node, state) {
   // extract the string values of keys to remove
   const keysToRemove = new Set(
     toRemovePropTypes.arguments[0].elements
-      .map(keyToRemove =>
+      .map((keyToRemove) =>
```

Comment:
`prettier` added parentheses, @BSFishy can you check if they were supposed to be added? If they shouldn't, I'll revert the `prettier` changes

Reply:
I just enabled `Prettier` in the settings of my `IDE`, it seems that `Prettier` was not applied to some files in the script folder before, f.e.
```
scripts/babel/fetch-i18n-strings.js
scripts/babel/react-docgen-typescript.js
scripts/eslint-plugin/forward_ref_display_name.js
scripts/eslint-plugin/i18n.js
```
and etc.

<img width="903" alt="Screenshot 2023-05-29 at 09 36 46" src="https://github.com/opensearch-project/oui/assets/40265277/cc367fc3-2eda-4972-91fe-3d1da7103c4e">

Reply:
i am glad to see you are using a real IDE... unlike some people... 😉 at Ashwin.